mmv1/products/networkservices/Gateway.yaml

```diff
@@ -130,3 +183,34 @@ properties:
     description: |
       A fully-qualified ServerTLSPolicy URL reference. Specifies how TLS traffic is terminated.
       If empty, TLS termination is disabled.
+  - !ruby/object:Api::Type::Array
+    name: 'addresses'
+    description: |
+      Zero or one IPv4-address on which the Gateway will receive the traffic. When no address is provided,
+      an IP from the subnetwork is allocated This field only applies to gateways of type 'SECURE_WEB_GATEWAY'.
+      Gateways of type 'OPEN_MESH' listen on 0.0.0.0.
+    item_type: Api::Type::String
+  - !ruby/object:Api::Type::String
+    name: 'subnetwork'
+    description: |
+      The relative resource name identifying the subnetwork in which this SWG is allocated.
+      For example: `projects/*/regions/us-central1/subnetworks/network-1`.
+      Currently, this field is specific to gateways of type 'SECURE_WEB_GATEWAY.
+  - !ruby/object:Api::Type::String
+    name: 'network'
+    description: |
+      The relative resource name identifying the VPC network that is using this configuration. 
+      For example: `projects/*/global/networks/network-1`. 
+      Currently, this field is specific to gateways of type 'SECURE_WEB_GATEWAY'.
```

Comment:
Do `addresses`, `subnetwork`, `network` support in-place update? If so, can we add an update test? Otherwise, we need to add `immutable`.

Reply:
Done